src/fsverity/ioctl.rs

```diff
@@ -53,7 +66,13 @@ pub fn fs_ioc_enable_verity<F: AsFd, H: FsVerityHashValue>(fd: F) -> std::io::Re
                 sig_ptr: 0,
                 __reserved2: [0; 11],
             }),
-        )?;
+        ) {
+            Err(e) if e.raw_os_error() == ENOTTY => {
```

Comment:
It should be possible to match directly on `Err(rustix::io::Errno::ENOTTY)` here, no?

Reply:
Indeed, silly me! Thanks and I simplified this now as suggested.